### PR TITLE
Update `Connect Kubectl` documentation

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
This PR aims to enhance the `Connect Kubectl` documentation

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc user
Enhanced the `Connect Kubectl` documentation
```

### CODE REVIEW

File: docs/usage/connect-kubectl.md [hunk 1]
```diff
@@ -1,54 +1,120 @@
-# Connect kubectl
+# Connect Kubectl
 
 In Kubernetes, the configuration for access to your cluster is a format known as `kubeconfig` that is stored as a file. It contains details such as cluster API server addresses and access credentials or a command to get the access credential from a `kubectl` credential plugin. In general, treat a `kubeconfig` as sensitive data. Tools like `kubectl` use the `kubeconfig` to connect and authenticate to a cluster and perform operations on it.
```
Comment (grolu) [inline docs/usage/connect-kubectl.md:3]:
```suggestion
In Kubernetes, the configuration for accessing your cluster is in a format known as `kubeconfig`, which is stored as a file. It contains details such as cluster API server addresses and access credentials or a command to obtain access credentials from a `kubectl` credential plugin. In general, treat a `kubeconfig` as sensitive data. Tools like `kubectl` use the `kubeconfig` to connect and authenticate to a cluster and perform operations on it.
```

File: docs/usage/connect-kubectl.md [hunk 2]
```diff
@@ -1,54 +1,120 @@
...
+2. Choose `CLUSTERS` and select your cluster in the list.
+3. Choose the `Show information about gardenlogin` info icon and follow the configuration hints.
+
+Note: Use the [previously downloaded](#download-kubeconfig-for-the-garden-cluster) kubeconfig for the `Garden` cluster as the kubeconfig path. Do not use the `gardenlogin` `Shoot` cluster kubeconfig here
+
+<img width="700" src="../images/03-gardenlogin-info.png">
+
+### Download and Setup Kubeconfig for a `Shoot` Cluster
+
+The `gardenlogin` kubeconfig for the `Shoot` cluster can be obtained in different ways:
```
Comment (grolu) [inline docs/usage/connect-kubectl.md:52]:
```suggestion
The `gardenlogin` kubeconfig for the `Shoot` cluster can be obtained in various ways:
```

File: docs/usage/connect-kubectl.md [hunk 3]
```diff
@@ -1,54 +1,120 @@
...
+1. Copy the `gardenctl target` command from the dashboard
+
+    <img width="1000" src="../images/copy-target-cluster-command.png">
+2. Paste and run the command in the terminal application, for example:
+```bash
+$ gardenctl target --garden landscape-dev --project core --shoot mycluster
+Successfully targeted shoot "mycluster"
+```
+
+Your `KUBECONFIG` env variable is now pointing to the current target (see also with `gardenctl target view -o yaml`). You can now run `kubectl` commands against your `Shoot` cluster.
```
Comment (grolu) [inline docs/usage/connect-kubectl.md:70]:
```suggestion
Your `KUBECONFIG` env variable is now pointing to the current target (also visible with `gardenctl target view -o yaml`). You can now run `kubectl` commands against your `Shoot` cluster.
```

File: docs/usage/connect-kubectl.md [hunk 4]
```diff
@@ -1,54 +1,120 @@
...
 ```
+The command connects to the cluster and list its namespaces.
+
+##### KUBECONFIG Env Var not Setup Correctly
+
+If your `KUBECONFIG` env variable does not point to the current target, you will see the following message after running the `gardenctl target` command:
+```bash
+WARN The KUBECONFIG environment variable does not point to the current target of gardenctl. Run `gardenctl kubectl-env --help` on how to configure the KUBECONFIG environment variable accordingly
+```
+In this case you would have to run the following command (assuming `bash` as your current shell). For other shells, consult the [gardenctl kubectl-env --help](https://github.com/gardener/gardenctl-v2/blob/master/docs/help/gardenctl_kubectl-env.md) documentation.
```
Comment (grolu) [inline docs/usage/connect-kubectl.md:83]:
```suggestion
In this case you would need to run the following command (assuming `bash` as your current shell). For other shells, consult the [gardenctl kubectl-env --help](https://github.com/gardener/gardenctl-v2/blob/master/docs/help/gardenctl_kubectl-env.md) documentation.
```

File: docs/usage/connect-kubectl.md [hunk 5]
```diff
@@ -1,54 +1,120 @@
...
+<img width="700" src="../images/download-garden-cluster-kubeconfig.png">
+
+##### Configure Gardenlogin
+
+Configure gardenlogin by following the installation instruction on the dashboard:
+1. Select your project from the dropdown on the left
+2. Choose `CLUSTERS` and select your cluster in the list.
+3. Choose the `Show information about gardenlogin` info icon and follow the configuration hints.
+
+Note: Use the [previously downloaded](#download-kubeconfig-for-the-garden-cluster) kubeconfig for the `Garden` cluster as the kubeconfig path. Do not use the `gardenlogin` `Shoot` cluster kubeconfig here
```
Comment (grolu) [inline docs/usage/connect-kubectl.md:46]:
```suggestion
>Note: Use the [previously downloaded](#download-kubeconfig-for-the-garden-cluster) kubeconfig for the `Garden` cluster as the kubeconfig path. Do not use the `gardenlogin` `Shoot` cluster kubeconfig here
```

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Review by grolu — APPROVED** [review]
/lgtm

**Pull request merged**